packages/playwright-test/src/runner.ts

```diff
@@ -448,7 +448,7 @@ export class Runner {
     await this._runAndReportError(async () => {
       for (const internalGlobalSetup of this._internalGlobalSetups)
         internalGlobalTeardowns.push(await internalGlobalSetup());
-      webServer = config.webServer ? await WebServer.create(config.webServer, this._reporter) : undefined;
```

Comment:
why not create a WebServer multiplexer aka. WebServers?

Reply:
Felt pre-mature given the impl without the class is just a few lines: https://github.com/microsoft/playwright/pull/13007/files#diff-acef657c29c494a2b26e391d22e5efec3513e9273e465fceb61e62d5cd91b264R41